### DISCUSSION TITLE
Undetected-Chromedriver alternative

### DISCUSSION THREAD

**Discussion by ttlns** [FlareSolverr/FlareSolverr#863] [Ideas]
@ngosang 
As you're awaiting the new implementation for  `undetected-chromedriver`, you might have a look at [Selenium-Driverless](https://github.com/kaliiiiiiiiii/Selenium-Driverless).
I assume that `undetected-chromedriver` will be using some similar implementation

It still might need some development tho.

**Comment by NCLnclNCL** [comment]
Never use this 😂😂, i saw a chromedriver sharer who passed the cloudflare dc so there's no reason why it can't be patched
